test/cpp/qps/server_async.cc

```diff
@@ -166,8 +100,16 @@ class AsyncQpsServerTest {
         return;
       }));
     }
-    while (!got_sigint) {
-      std::this_thread::sleep_for(std::chrono::seconds(5));
+  }
+  ~AsyncQpsServerTest() {
+    server_->Shutdown();
+    srv_cq_.Shutdown();
```

Comment:
Is it not necessary to clear out the srv_cq_ as was being done earlier with Next calls?